ipaplatform/base/services.py

```diff
@@ -483,10 +483,7 @@ def remove(self):
 
 # Objects below are expected to be exported by platform module
 
-def base_service_class_factory(name):
```

Comment:
The dummy function was required to get pylint passing. I don't see any harm in keeping it.